_std/defines/jobs.dm

```diff
@@ -39,4 +39,5 @@
 #define MGA_SHIPPING "Shipping Alert"
 #define MGA_CARGOREQUEST "Cargo Request"
 #define MGA_CRISIS "Crisis Alert"
-#define MGA_RADIO "Radio Alert"
\ No newline at end of file
+#define MGA_RADIO "Radio Alert"
+#define MGA_TRACKING "Tracking Alert"
```

Comment:
the only things that were in _std seem good so my approving review is now done :smile: (idk enough abt this code to actually review it)